docs/CONTRIBUTORS.rst

```diff
@@ -113,9 +113,8 @@ CI/CD will check that new TUF code is formatted with `black
 Auto-formatting can be done on the command line:
 ::
 
-    $ # TODO: configure black and isort args in pyproject.toml (see #1161)
-    $ black --line-length 80 tuf/api
-    $ isort --line-length 80 --profile black -p tuf tuf/api
+    $ black tuf/api tuf/ngclient
+    $ isort tuf/api tuf/ngclient
```

Comment:
Let's also format and lint the scripts in the newly added `examples` directory. (see #1685)

Reply:
could just do this to avoid the maintenance here:

```suggestion
    $ black <filename>
    $ isort <filename>
```

Reply:
The whole `black`/`isort` file-based case handling should hopefully soon be irrelevant, when we remove the legacy code.